packages/zowe-explorer-api/src/profiles/ProfilesCache.ts

```diff
@@ -38,19 +43,38 @@ export class ProfilesCache {
         return require("@zowe/secrets-for-zowe-sdk").keyring;
     }
 
+    /**
+     * Adds profile type meta-data to array for future use with team configuration actions,
+     * ie. Zowe Explorer uses upon extender registration in VS Code.
+     *
+     * @param {imperative.ICommandProfileTypeConfiguration[]} extendermetadata Profile type meta-data used with Zowe team configurations
+     *
+     * @returns {void}
+     */
     public addToConfigArray(extendermetadata: imperative.ICommandProfileTypeConfiguration[]): void {
+        // Disabling deprecation warning in method as to not break extenders.
+        // Will need to continue updating array in case of use of the deprecated constant.
         extendermetadata?.forEach((item) => {
-            const index = this.profileTypeConfigurations.findIndex((ele) => ele.type == item.type);
+            const index = ProfilesCache.sessionProfileTypeConfigurations.findIndex((ele) => ele.type == item.type);
             if (index !== -1) {
-                this.profileTypeConfigurations[index] = item;
+                // eslint-disable-next-line deprecation/deprecation
+                ProfilesCache.sessionProfileTypeConfigurations[index] = this.profileTypeConfigurations[index] = item;
             } else {
+                // eslint-disable-next-line deprecation/deprecation
                 this.profileTypeConfigurations.push(item);
+                ProfilesCache.sessionProfileTypeConfigurations.push(item);
             }
         });
     }
 
+    /**
+     * Returns an array of Zowe Explorer registered profile types and core Zowe supported profile types meta-data ,
+     * used with Zowe team configuration actions.
+     *
+     * @returns {imperative.ICommandProfileTypeConfiguration[]}
+     */
     public getConfigArray(): imperative.ICommandProfileTypeConfiguration[] {
-        return this.profileTypeConfigurations;
+        return ProfilesCache.sessionProfileTypeConfigurations;
```

Comment:
This change won't directly impact us, but I do wonder if this affects extenders who extend `ProfilesCache` for their own use cases...

Since subclasses of `ProfilesCache` are likely uncommon, and the change could be argued as a bug fix, I'm not specifically requesting changes for this - just something that came to mind 😋

Reply:
in the sense of extender, this change will fix a bug where they were not receiving the registered extender types just the core types we add in the method.